pkgs/development/libraries/nyan/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, clang_14
+, cmake
+, flex
+}:
+
+stdenv.mkDerivation {
+  pname = "nyan";
+  version = "0.2.0";
```

https://github.com/SFTtech/nyan/tags
The most recent tag is 0.2, not 0.2.0

```suggestion
  version = "0.2";
```